### ISSUE TITLE
RC 2.6.0: NS cancels temporary target 

### ISSUE THREAD

**Issue by chriske64** [MilosKozak/AndroidAPS#2281]
As of the new version, NS always cancels my TT. Maybe I have poor NS settings but I haven't had the problem before. When I pause NS then it works fine.

**Comment by MilosKozak** [comment]
what do you mean by NS? Nightscout itsefl doesn't do anything

**Comment by chriske64** [comment]
Okay, at least the Temp Target is canceled and if I stop the connection to NS it is not. Maybe it has to do with the fact that I don't upload local profile changes to NS or something like that.

**Comment by MilosKozak** [comment]
sorry, it works for me well
if you are able reproduce this issue create new issue with step by step how to make this bug appear.
include log files pls

**Comment by chriske64** [comment]
I can easily reproduce it. Indeed if I am connected to Nightscout I am not able to create a Temporary Target. More precisely, if I create one, it gets canceled shortly after. I have attached the log files. In the Log file I have switched my TT to 111 and at 14.20 and then at 14.22 the Temporary Target canceled and the target BG was back to 100.

[TT_111.log](https://github.com/MilosKozak/AndroidAPS/files/3975129/TT_111.log)
